src/current/_includes/common/upgrade/finalize-cloud.md

```diff
@@ -0,0 +1,10 @@
+To finalize a major-version upgrade manually,
+
+1. Click the cluster's name to open the **Cluster Details** page.
+1. At the top of the page, click **Finalize**.
+
+    When finalization begins, a series of migration jobs run to enable certain types of features and changes in the new major version that cannot be rolled back. These include changes to system schemas, indexes, and descriptors, and enabling certain types of improvements and new features. These temporary limitations are listed in the release notes for the new major version. Until the upgrade is finalized, these features and functions will not be available and the command `SHOW CLUSTER SETTING version` will continue to report the previous major version.
```

Comment:
The "These temporary limitations" sentence feels like it belongs in its own paragraph, with a statement that there _are_ temporary limits.

Reply:
Done